templates/index.md.tmpl

```diff
@@ -10,11 +10,24 @@ description: |-
 
 The provider can be used to interact with [Juju][0] - a model-driven Operator Lifecycle Manager (OLM) by Canonical.
```

Comment:
I'd again drop the OLM wording. Also, I'd use the official definition from, e.g., here: https://github.com/juju/juju